site/en/blog/prerender-pages/index.md

```diff
@@ -0,0 +1,263 @@
+---
+# Required
+layout: 'layouts/blog-post.njk'
+title: Prerender pages in Chrome for instant page navigations
+description: |
+ ??
+authors:
+ - tunetheweb
+date: 2022-11-28
+#updated: 2022-11-28
+hero: image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/eohdiqaZlxnWen7TT66M.jpg
+alt: City road at dusk with a long exposure of car lights giving impression of speed
+tags:
+ - performance
+ - chrome-108
+---
+
+As [announced previously](https://web.dev/speculative-prerendering/), the Chrome team has been working on options to bring back full prerendering of future pages that a user is likely to navigate to. This modern reboot of prerendering", and from there on simply prerendering will start rolling out from Chrome 108.
```

Comment:
Looks like you have an hanging quotation mark. Not sure where you intended to have the open quote start, so just letting you know here.